### ISSUE TITLE
Maybe we could get nix.org?

### ISSUE THREAD

**Issue by iFreilicht** [nix-rfc-canonical-domain/rfcs#8]
It's mentioned specifically in the RFC as being potentially too costly, but we don't know if we don't try.

According to [WHOIS](https://www.whois.com/whois/nix.org), the domain was registered in 2000 and has either been sold recently or is currently owned by [MarkMonitor](https://www.markmonitor.com/domains-for-sale/top-level-domains), a domain broker. [They also own(ed) nix.com](https://www.whois.com/whois/nix.com).

The domain doesn't seem to currently be available on their website, but we could contact them about it. I would gladly do it, but I fear that with my name not showing up on the Nix homepage anywhere and me not having an @nixos.org email address (does anyone have one of those?), they would not take my request seriously.

Additional benefit of nix.org that are not mentioned in the RFC yet:

* Like gnu.org or eff.org, shows that Nix is a serious and long-lasting project
* It is very obviously "correct", I believe there would be much less discussion around it
* .org is most commonly used by non-profits and open-source projects, shows that Nix is not commercial software, but free and open
* Having a third domain to redirect to allows for a zero-conflict transition. all `nix.dev` links can forward to `docs.nix.org`, all `nixos.org` links can redirect to `nix.org`

**Comment by iFreilicht** [comment]
Ah, someone linked [this comment from about a year ago](https://github.com/NixOS/nix.dev/issues/290#issuecomment-1198629843) that says:

> @edolstra: @grahamc contacted markmonitor.com (https://github.com/NixOS/nixos-homepage/issues/633#issuecomment-841634501) and it appears they would take tens of thousands USD for the transfer

Frustrating, though @infinisil and @thufschmitt already argued that it might be worth it. The foundation does have significant reserves right now and could indeed afford it, though we should of course get the best possible deal.

**Comment by infinisil** [comment]
Yeah this might be worth another thought, `nix.org` would really be the obvious choice if we could.

**Comment by infinisil** [comment]
I just contacted Graham for more detailed info:
- nixos.com (we're not very interested in this one, but still good to know)
  - Contacted them a few years ago when they wanted 20k EUR
  - Contacted them again, now wanting 250k EUR
- nix.org
  - Wasn't contacted, but we're unlikely to get it, it's probably owned by "Prestige Consumer Healthcare, manufacturer of Nix head lice medicine."
- nix.com
  - Wasn't contacted, but expecting it to be over 1M USD

So it's not looking good, but I think we should still try. I might follow up on this after talking to the foundation.

**Comment by iFreilicht** [comment]
Thank you so much!

> * nixos.org (we're not very interested in this one, but still good to know)

Was there some mixup? nixos.org is the one we already have.

**Comment by infinisil** [comment]
Ahh yes, should be nixos.com (the adult one), updated the comment

**Comment by edolstra** [comment]
I think `nix.dev` is better than `nix.org` or `nix.com`. Our homepage primarily promotes Nix as a development tool (e.g. for dev shells). And there is no "Nix" organisation.

**Comment by infinisil** [comment]
It should be the "Nix Foundation" imo :) https://github.com/NixOS/foundation/issues/77

But I don't think just because the _focus_ is on development means that the domain should reflect that. Nix can be used for more than just development, we shouldn't disregard that fact.

**Comment by iFreilicht** [comment]
> Our homepage primarily promotes Nix as a development tool

Right now it does, but Nix is more than that and could be advertised as such.

> And there is no "Nix" organisation.

At the very least there is the NixOS organization here on github and the adjacent nix-community org. nix.org would promise a single entry point for the whole ecosystem, not just the developer tool.

**Comment by mightyiam** [comment]
I've tried following a process to acquire brokerage service for nix.org at https://sedo.com but it's EUR 99. Please, start such a process (with some provider) and see where the negotiations lead.

I reached sedo.com via their partnership with porkbun.com. Since I've been enjoying porkbun.com, perhaps sedo.com, by affiliation, provide good service 🤷 

@hsjobeki @tomberek @NixOS/foundation
